archivist_samples/software_bill_of_materials/README.md

```diff
@@ -11,35 +11,21 @@ This sample shows how to quickly get started with integrating your build and SBO
 
 ## Pre-requisites
 
-Python 3.6 and later versions are supported.
+* Python 3.6 and later versions are supported.
 
-Install the [RKVST Python package](https://pypi.org/project/jitsuin-archivist/ "PyPi package page")
+* Install the [RKVST Python package](https://pypi.org/project/jitsuin-archivist/ "PyPi package page")
```

Comment:
Either this is not needed (because the user has already installed the archivist-samples package) or should be changed to installing the archivust-samples package...

Reply:
Fixed